docs/reference/glossary.md

```diff
@@ -0,0 +1,161 @@
+
+Note: if used as a proper noun, capitalize the term. For example, "I am mining on Testnet."
+
+## Tipset
+
+Filecoin utilizes a novel [blockchain](#blockchain) variant called a [_tipset_](https://filecoin.io/blog/tipsets-family-based-approach-to-consensus/). Tipsets are less strict chains than directed, acyclic graphs; this construction allows multiple blocks to be mined at the same [height](#block-height).
+
+## Verified client
+
+To further incentivize the storage of "useful" data over simple [capacity commitments](#capacity-commitment), [storage miners](#storage-miner) have the additional opportunity to compete for special [deals](#deal) offered by [verified clients](#verified-client). Such clients are certified with respect to their intent to offer deals involving the storage of meaningful data, and the power a storage miner earns for these deals is augmented by a multiplier.
+
+## Window Proof-of-Spacetime (WindowPoSt)
+
+_Window Proof-of-Spacetime_ (WindowPoSt) is the the mechanism by which the commitments made by [storage miners](#storage-miner) are audited. It sees each 24-hour period broken down into a series of windows. Correspondingly, each storage miner’s set of pledged [sectors](#sector) is partitioned into subsets, one subset for each window. Within a given window, each storage miner must submit a [Proof-of-Spacetime](#proof-of-spacetime-post) for each sector in their respective subset. This requires ready access to each of the challenged sectors, and will result in a [zk-SNARK-compressed](#zero-knowledge-succinct-non-interactive-argument-of-knowledge-zk-snark) proof published to the Filecoin [blockchain](#blockchain) as a [message](#message) in a [block](#block). In this way, every sector of [pledged storage](#pledged-storage) is audited at least once in any 24-hour period, and a permanent, verifiable, and public record attesting to each storage miner’s continued commitment is kept.
+
+The Filecoin network expects constant availability of stored data. Failing to submit WindowPoSt for a sector will result in a [fault](#fault), and the storage miner supplying the sector will be [slashed](#slash).
+
+## Winning Proof-of-Spacetime (WinningPoSt)
+
+_Winning Proof-of-Spacetime_ (WinningPoSt) is the mechanism by which [storage miners](#storage-miner) are rewarded for their contributions to the Filecoin network. At the beginning of each [epoch](#epoch), a small number of storage miners are [elected](#election) to each mine a new [block](#block). As a requirement for doing so, each miner is tasked with submitting a compressed [Proof-of-Storage](#proof-of-storage-post) for a specified [sector](#sector). Each elected miner who successfully creates a block is granted filecoin, as well as the opportunity to charge other Filecoin participants fees to include [messages](#message) in the block.
```

Comment:
```suggestion
_Winning Proof-of-Spacetime_ (WinningPoSt) is the mechanism by which [storage miners](#storage-miner) are rewarded for their contributions to the Filecoin network. At the beginning of each [epoch](#epoch), a small number of storage miners are [elected](#election) to each mine a new [block](#block). As a requirement for doing so, each miner is tasked with submitting a compressed [Proof-of-Storage](#proof-of-storage-post) for a specified [sector](#sector). Each elected miner who successfully creates a block is granted Filecoin, as well as the opportunity to charge other Filecoin participants fees to include [messages](#message) in the block.
```

Reply:
Let's use FIL to represent the token, as proposed in #215.